src/context/UserContextProvider.tsx

```diff
@@ -163,6 +163,7 @@ export const UserContextProvider: React.FC = (props): JSX.Element => {
       // don't support .domain in browsers while setting cookies
       // include the leading dot only for "real" domains
       domain: hostname.includes('.') ? `.${hostname}` : hostname,
+      sameSite: 'lax',
```

Comment:
Is this set just to cover older Firefox versions because I see that now default value is `lax` anyway. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite

Reply:
Unless I am misinterpreting https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#browser_compatibility looks like firefox has does not set `sameSite` by default as it is an optional setting. Also looks like it is also disabled in safari so might still be useful anyway?